<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StereoUI</class>
 <widget class="QWidget" name="StereoUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="eye_s_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Eye spacing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="eye_spacing">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="value">
        <double>0.065000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="focal_length_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Convergence length</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="focal_length">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>10000.989999999999782</double>
       </property>
       <property name="value">
        <double>2.500000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="pixel_shift_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pixel shift</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="pixel_shift">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="screen_width_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Screen width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="screen_width">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>( if &lt;= 0, will use default)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="left_color">
       <property name="text">
        <string>Left Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="right_color">
       <property name="text">
        <string>Right Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="color_label">
       <property name="text">
        <string>(Only  valid for anaglyphic)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Technique</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QRadioButton" name="anaglyph">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Anaglyph</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="passive">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Passive</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="active">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Active</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="enable">
       <property name="text">
        <string>Enable stereo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disable">
       <property name="text">
        <string>Disable stereo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
